CHARTER.md

```diff
@@ -0,0 +1,192 @@
+
+#### Section 4.2.1. Elections
+
+Leadership roles in the webpack project will be peer elected representatives of the community.
+
+For election of persons (such as the TSC Chairperson), a multiple-candidate method should be used, such as:
+
+* [Condorcet][] or
+* [Single Transferable Vote][]
+
+Multiple-candidate methods may be reduced to simple election by plurality when there are only two candidates for one 
+position to be filled. No election is required if there is only one candidate and no objections to the candidate's 
+election. Elections shall be done within the projects by the Collaborators active in the project.
+
+The TSC voting members will elect from amongst voting TSC members a TSC Chairperson to work on building an agenda for 
+TSC meetings and a OpenJS Cross Project Council (CPC) voting member to represent the TSC in the OpenJS Foundation for a 
+term of one year. The Chair and voting CPC member may be (but are not required to be) the same person. The TSC shall 
+hold annual elections to select a TSC Chairperson and voting CPC member; there are no limits on the number of terms a 
+TSC Chairperson or voting CPC member may serve.
+
+#### Section 4.2.2. Voting
+
+For internal project decisions, Collaborators shall operate under Lazy Consensus. The TSC voting members shall establish 
+appropriate guidelines for implementing Lazy Consensus (e.g. expected notification and review time periods) within the 
+development process.
+
+The TSC voting members follow a [Consensus Seeking][] decision making model. When an agenda item has appeared to reach a 
+consensus the moderator will ask "Does anyone object?" as a final call for dissent from the consensus.
+
+For all votes, a simple majority of all TSC voting members for, or against, the issue wins. A TSC voting member may 
+choose to participate in any vote through abstention.
+
+All changes to this charter must be approved by the CPC.
+
+#### Section 4.2.3. Project Roles
+
+The webpack git repository is maintained by the TSC and additional Collaborators who are added by the TSC voting members 
+on an ongoing basis.
+
+Individuals making significant and valuable contributions are made Collaborators and given commit-access to the project.
+These individuals are identified by the TSC and their addition as Collaborators is discussed during a TSC meeting. 
+Modifications of the contents of the git repository are made on a collaborative basis as defined in the development 
+process.
+
+Collaborators may opt to elevate significant or controversial modifications, or modifications that have not found 
+consensus to the TSC for discussion by assigning the `tsc-agenda` tag to a pull request or issue. The TSC voting members 
+should serve as the final arbiter where required. The TSC will maintain and publish a list of current Collaborators, as 
+well as a development process guide for Collaborators and Contributors looking to participate in the development effort.
```

Comment:
Don't put implementation details in the charter
```suggestion
Collaborators may opt to elevate significant or controversial modifications, or modifications that have not found 
consensus to the TSC for discussion. The TSC voting members 
should serve as the final arbiter where required. The TSC will maintain and publish a list of current Collaborators, as 
well as a development process guide for Collaborators and Contributors looking to participate in the development effort.
```

Reply:
We removed this section